bitnami/thanos/values.yaml

```
@@ -714,13 +714,15 @@ query:
   ## @param query.autoscaling.maxReplicas Maximum number of Thanos Query replicas
   ## @param query.autoscaling.targetCPU Target CPU utilization percentage
   ## @param query.autoscaling.targetMemory Target Memory utilization percentage
+  ## @param query.autoscaling.targetPodMetrics Target Pod metric name and AverageValue
```

Given a specific object array structure is expected, we should improve the parameter definition and add some example:
```suggestion
  ## @param query.autoscaling.targetPodMetrics [array] List of custom pod metrics name and averageValue to evaluate for scaling
  ## e.g.
  ## targetPodMetrics:
  ## - name: thanos_query_range_query_duration
  ##   averageValue: 2
```